src/radiation/radiation.cpp

```diff
@@ -95,11 +95,13 @@ void ablate::radiation::Radiation::Setup(const solver::Range& cellRange, ablate:
 
     /** Declare some information associated with the field declarations */
     PetscReal* coord;                   //!< Pointer to the coordinate field information
+    PetscInt* index;                    //!< Pointer to the cell index information
```

Comment:
The index turns into a pointer accessing the swarm field now. A lot of the changes regarding the cell index are just to add the particle index on the end.